### ISSUE TITLE
Padding cells on auto resize

### ISSUE THREAD

**Issue by salimnanji** [free-jqgrid/jqGrid#188]
I've noticed that when using autoresizing, the text is very compact in the cells. Have you thought about adding some padding when you resize?

See the two fiddles below with the compacted grid, and what I whipped up to have some padding

Grid now: http://jsfiddle.net/LLx2hq54/9/
Grid with padding on resize:  http://jsfiddle.net/LLx2hq54/8/

**Comment by OlegKi** [comment]
Thank you for your suggestion!

I think that the main problem of free jqGrid now is missing separate documentation and the collection of demos, which could be permanently extended. I'm working currently on this.

Back to you suggestion. I see that the default padding `2px` used in jqGrid is not good enough in many cases of the usage. I wrote the code of `getAutoResizableWidth` (which are used in `autoResizeColumn`, `autoResizeAllColumns` and `autoresizeOnLoad: true`) so that **one can increase the padding in the grid in CSS** and the padding will be used by jqGrid automatically. For example one can specify
```css
.ui-jqgrid .ui-jqgrid-htable th {
	padding: 0 .5em 0 .5em;
}
.ui-jqgrid tr.jqgrow > td,
.ui-jqgrid tr.jqgroup > td,
.ui-jqgrid tr.jqfoot > td,
.ui-jqgrid tr.jqfoot > td,
.ui-jqgrid tr.jqgfirstrow > td {
	padding: 0 .5em 0 .5em;
}
```
to use `0.5em` right and left padding and `0` for top and bottom padding. You can see the results on the demo http://jsfiddle.net/OlegKi/LLx2hq54/10/. I think it's what you need to do.

It's important that one should increase the padding not only for `.ui-jqgrid tr.jqgrow > td` and `.ui-jqgrid .ui-jqgrid-htable th`, but for **`.ui-jqgrid tr.jqgfirstrow > td`** too. The row `tr.jqgfirstrow` is the first row of the table (it has the height 0 and thus it's not visible), which will be used to calculate the width of all column (because of `table-layout: fixed` used for tables).

**Comment by salimnanji** [comment]
Thank you Oleg! That's exactly what I need.

Any reason you specified it that way instead of just using:

.ui-jqgrid .ui-jqgrid-htable th,
.ui-jqgrid tr.jqgrow > td,
.ui-jqgrid tr.jqgroup > td,
.ui-jqgrid tr.jqfoot > td,
.ui-jqgrid tr.jqfoot > td,
.ui-jqgrid tr.jqgfirstrow > td {
    padding: 0 .5em 0 .5em;
}

**Comment by OlegKi** [comment]
The only reason was because I did many things parallel and just made copy and paste of the parts from `ui.jqgrid.css` without combining of all together. For example you did the same thing and so you included two identical rules for `.ui-jqgrid tr.jqfoot > td` like I did before and like it's exist in the current `ui.jqgrid.css`. :-) I posted the corresponding fix to `ui.jqgrid.css` just now (see [here](https://github.com/free-jqgrid/jqGrid/commit/564fa199e82858709b338f07fe76e5db5f5a7b9b)).
